### ISSUE TITLE
God awful nuclear operatives and heads

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#14842]
I believe we should have a minimum of a month of being on the server to be able to play nuclear operatives, we've had numerous dumbasses screw over their team by buying stupid things or not knowing how to trigger their e-implant 
It's not very fun as a nuke op and it's boring when you're crew and a nuke op that doesn't know how to death gasp spoonfeeds you a holochip after being arrested
It's different from cult because you can just be quiet and not do stupid things and still be useful for the cult if you arrive at the summoning plus it's easier to handhold new cultists or ask yred for a cult school

Also head roll player age should really be increased to prevent a bit more people from being horrible at it and getting banned
VOX ops isn't a problem because it's a shit game mode either way

**Comment by Arthurdentist** [comment]
one of the things that is aggravating about nuke cop rounds  from an admin perspective is that it is very difficult to 'fix' them. You can create new cops but it can take time which puts people off, meaning some of the people who got screwed will go braindead. It'd be nice if there were some neat buttons for this, along the lines of the 'DRESS-UP, SEND TO STATION, LET CHOOSE NAME' buttons available for admins creating wizards. 

inb4 antag datums.

**Comment by ghost** [comment]
I've had admins just take out the player's mind and put a ghost in

**Comment by ghost** [comment]
Though only once because I ahelped one of our nuke buddies lagging too much

**Comment by Arthurdentist** [comment]
I'm thinking more along the lines of when bodies have been gibbed. I'd like to be able to create a body, put a guy in it, toggle him as nuke op then just press another button to 'dress him up'.

**Comment by Shadowmech88** [comment]
There should probably be a minimum playtime on all antags, at least a week or so.

**Comment by ghost** [comment]
I thought admins got a "make a nuclear operative" button

**Comment by ghost** [comment]
>all antags
Probably exclude traitor

**Comment by Fruktlimpa** [comment]
Now I have only been admin on my own server but doesn't the antag madness verb have a Nuclear operative option?
Sure, I think it spawns extra items which might be the issue.

**Comment by Killette2** [comment]
>It's a "LET'S RUSH" and have no plan episode

Reason I have nuke ops turned off

**Comment by FudgePucker** [comment]
I still remember the round where I got teleported in on top of by my teammate and RPGed in the face after I got the captain taken down single handedly after an admin on my team told me I was the last one left.

**Comment by Killette2** [comment]
The admin lied

**Comment by FudgePucker** [comment]
I think he just didn't know, the last teammate dragged the nuke onto the station, then went back to the ship and then though the teleporter, instead of joining the rest of the team on the suicide charge. The admin was in the suicide charge after the first idiot blew the plan early. After I got the PM I asked the captain for all access, while my ID was off so instead of being voice changed to the Assistant I was body doubling as it was my obvious agent name, THE ABSOLUTE MAD MAN gave me all access, so I rushed to the TEG, put a canister labeled air on the hot loop connected, dragged it back to captains room where the captain was, popped it with a welder, I was celebrating when I saw the captain went down, right then my last teammate alive teleported and RPG'd me in the face.
